src/client/containers/EventsPage/EventsPage.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState, useEffect } from 'react';
+import { AppHeader } from '../../components/Appheader/AppHeader.component';
+import Footer from '../../components/footer/footer';
+import './EventsPage.styles.css';
+import Header from '../../components/Heading/Heading';
+import {
+  CardExampleWithHYF,
+  ExampleWithElastic,
+  ExampleWithZendesk,
+} from '../../components/CardWithEventsOrStudyGroups/CardWithEventsOrStudyGroups.stories';
+
+function EventsPage() {
+  const [events, setEvents] = useState([]);
+
+  useEffect(() => {
+    const getEvents = async () => {
+      const resp = await fetch('/api/events');
+      const eventsResult = await resp.json();
+      setEvents(eventsResult);
+    };
+    getEvents();
+  }, []);
+
+  return (
+    <div className="events-page">
+      <AppHeader id="app-header" />
+      <Header>Current Studygroup</Header>
+      <div className="group-for-class">Study groups for class15</div>
```

Comment:
```suggestion
      <div className="group-for-class">Study groups</div>
```
Remove the class since we don't know which class this user belongs to.